CHANGELOG.md

```diff
@@ -18,7 +18,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - WebConsole: The pipeline view now also shows a graph of memory utilization over time
   ([#610](https://github.com/feldera/feldera/pull/610))
-
+- WebConsole: Added way to delete rows when browsing the tables of a pipeline
+  ([#612](https://github.com/feldera/feldera/issues/612)).
+- WebConsole: Improved error reporting added the ability to open new github issues in the
```

Comment:
Improved error reporting: added the ability to open new github issues...

Is the health dashboard the dropdown view for each pipelline?

Reply:
health dashboard Is on home screen, will clarify